WeTransferPRLinter/Sources/WeTransferPRLinter/XCResultReporting/ResultItems/TestSummaries.swift

```diff
@@ -87,14 +104,27 @@ extension ActionTestSummaryGroup {
     var failedTestIdentifiers: Set<String> {
         subtests.failedTestIdentifiers.union(subtestGroups.failedTestIdentifiers)
     }
+
+    var retriedTestIdentifiers: Set<String> {
+        subtests.retriedTestIdentifiers.union(subtestGroups.retriedTestIdentifiers)
+    }
 }
 
 extension Array where Element == ActionTestMetadata {
-    var failedTestIdentifiers: Set<String> {
-        let successIdentifiers = Set<String>(filter { $0.testStatus == "Success" }.map { $0.identifier })
-        let failedIdentifiers = Set<String>(filter { $0.testStatus == "Failure" }.map { $0.identifier })
+    private var successIdentifiers: Set<String> {
+        Set<String>(filter { $0.testStatus == "Success" }.map { $0.identifier })
+    }
+    private var failedIdentifiers: Set<String> {
+        Set<String>(filter { $0.testStatus == "Failure" }.map { $0.identifier })
+    }
 
+    var failedTestIdentifiers: Set<String> {
         /// Substract success identifiers to filter out retried tests.
         return failedIdentifiers.subtracting(successIdentifiers)
     }
+
+    var retriedTestIdentifiers: Set<String> {
+        /// Tests that succeeded eventually intersect with failed tests.
```

Comment:
I think this header doc (as well as the one above) should ideally be placed above the `var`, so we can option-click on it in Xcode? :)

Reply:
I added this as a note to the implementation details only. The variable is documented a bit higher up the hierarchy